k8s/helm/component-patch-tpl.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: v1
+
+values:
+  images:
+    ces-control: cloudogu/k8s-ces-control:0.4.0
+
+patches:
+  values.yaml:
+    manager:
+      image:
+        repository: "{{ registryFrom .images.ces-control }}/{{ repositoryFrom .images.ces-control }}"
+        tag: "{{ tagFrom .images.ces-control }}"
```

```suggestion
    cesControl: cloudogu/k8s-ces-control:0.4.0

patches:
  values.yaml:
    manager:
      image:
        repository: "{{ registryFrom .images.cesControl }}/{{ repositoryFrom .images.cesControl }}"
        tag: "{{ tagFrom .images.cesControl }}"
```
the image-key should not have hyphens. 
Sorry, I missed it on the first run